cmd/outputs.go

```diff
@@ -62,17 +63,7 @@ func getAllOutputConstructors() (map[string]func(output.Params) (output.Output,
 			}
 			return newCollectorAdapter(params, influxc), nil
 		},
-		"kafka": func(params output.Params) (output.Output, error) {
-			conf, err := kafka.GetConsolidatedConfig(params.JSONConfig, params.Environment, params.ConfigArgument)
-			if err != nil {
-				return nil, err
-			}
-			kafkac, err := kafka.New(params.Logger, conf)
-			if err != nil {
-				return nil, err
-			}
-			return newCollectorAdapter(params, kafkac), nil
-		},
+		"kafka": func(p output.Params) (output.Output, error) { return kafka.New(p) },
```

Comment:
We should probably emit a deprecation warning here? We have a logger in `output.Params`, and this is called only once, so it should be very easy...

Reply:
That is a good idea. 

I will wait until we decide whether we will be going forward with this and where the repo will be put to make any changes though ;)

Reply:
also, post https://github.com/loadimpact/k6/issues/1921 just so we don't complicate it even more

Reply:
added it in 2504ac0